tools/fuzzers/fuzzer-json/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+if(HERMES_ENABLE_LIBFUZZER AND HERMES_ENABLE_LPMFUZZERS)
+    set(CMAKE_CXX_STANDARD 14)
+    set(CMAKE_CXX_STANDARD_REQUIRED ON)
+    set(HERMES_ENABLE_EH ON)
+
+    if (NOT EXISTS ${PROJECT_SOURCE_DIR}/tools/fuzzers/fuzzer-json/json.pb.cc)
+        message(FATAL_ERROR "Proto file is not compiled, run tools/fuzzers/fuzzer-json/compile_proto.sh")
+    endif()
```

Comment:
CMake can compile C++ files as part of the build. This shouldn't be necessary as a separate step.

Reply:
This is to compile the protobuf files. Do you want me to make it directly into cmake file instead of asking users to run scripts?

Reply:
Yes if possible we should compile this file as part of the build of the fuzzer tool.
It can be done like so:
```
add_executable(mytool ARGS...)
add_custom_command(
    OUTPUT  json.pb.cc
    COMMAND mytool ${CMAKE_CURRENT_BINARY_DIR}/json.pb.cc
    DEPENDS someInputFile.cpp.in
)
```
You can place the `add_executable` in the "external" directory so that we know it's building a third party tool for Hermes. It should also use `find_package` to avoid being built unless the LPM package is available.
If LPM is distributed with the generator tool available, you won't even need to build it at all

Reply:
If LPM isn't distributed via `apt`, then I would follow the instructions here:
https://github.com/google/libprotobuf-mutator#quick-start-on-debianubuntu
And complete the "install" step which will put them into the `/usr/local/include` path to find the headers
and libraries.
You can include a link to that in a message if the package isn't found on the system